master/getting-started/mesos/installation/dc-os/custom.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Customizing Calico-DC/OS
```

Comment:
This doesn't match the sidebar title for this doc (Custom Framework Install).

Would probably be better as "Configuring the Calico Universe Package"